CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 3.13.4 Baltic Porter (2020-01-13)
+
+## Features
+
+## Bug Fixes
+
+### Subscriptions
+ * custom products can't be consumed by hosts ([#28714](https://projects.theforeman.org/issues/28714), [9c28bc97](https://github.com/Katello/katello.git/commit/9c28bc97a553977269f2eeb5feed79370580a833), [eb233a31](https://github.com/Katello/katello.git/commit/eb233a310083413045bf63c634865ad4f3adf076))
```

Comment:
The issue has some closed (unmerged) PRs associated - can you remove then and regenerate the changelog?

Reply:
Needs another regenerate - the CP PR shouldn't be linked to the issue - I removed it for ya

Reply:
I may not understand this completely, but both of those commits are related. The first is a PR to address the original redmine issue, while the second commit it @jlsherrill 's PR to correct a VCR tape related to the same redmine issue.

When generating the CP from tool_belt, both commit SHA are indicated with the same description. Both commits can be picked without conflict, etc. Are you suggesting we drop one of those (the VCR update)? Let me know if I understand what you meant.

Reply:
My bad, I was misinterpreting what I was reading here. Changelog looks good!